src/utility/schoolUtil.ts

```diff
@@ -150,4 +116,22 @@ export class schoolUtil {
       })
     );
   };
+  public static getCurrMode(): MODES {
+    const api = ServiceConfig.getI().apiHandler;
+
+    if (!!api.currentMode) return api.currentMode;
+    const currMode = localStorage.getItem(CURRENT_MODE);
+    console.log(currMode);
+    if (!currMode) return MODES.PARENT;
+    const tempMode: MODES = MODES[currMode as keyof typeof MODES];
+    api.currentMode = tempMode;
+
+    if (tempMode === MODES.SCHOOL) return MODES.SCHOOL;
```

Comment:
why don't you just return tempMode here??

Reply:
there is no difference in tempmode in MODES.SCHOOL we can use any one of them if(tempMode is equal to MODES.SCHOOL